src/index.js

```diff
@@ -41,24 +41,29 @@ export default class Machine extends erx.Bus<State> {
   }
 
   goTo(next: State, data?: any): boolean {
+    const currEntries = this.transitions[this.state] || null;
```

Comment:
We should remove transitions object now that we have hooks per entry? The entries should come from the entries table (although i think we committed to renaming entries to "states")

Reply:
Yep the naming should change from transitions -> entries